rootfs/etc/services.d/pulseaudio/run

```diff
@@ -5,4 +5,4 @@
 export PULSE_STATE_PATH="/data/states"
 export LD_PRELOAD="/usr/local/lib/libjemalloc.so.2"
 
-exec pulseaudio --system -vvv --disallow-exit --exit-idle-time=-1 --disable-shm
+exec pulseaudio --system --disallow-exit --exit-idle-time=-1 --disable-shm
```

Comment:
```suggestion
log_level="$(bashio::supervisor.logging)"
if [ "${log_level}" == "critical" ]; then
    log_level="error"
fi
exec pulseaudio --system --disallow-exit --exit-idle-time=-1 --disable-shm "--log-level=${log_level}"
```
Think something like this will work fine. You don't actually need to set the value in the config file, providing the `log-level` parameter overrides that.

Reply:
According to the man page the log levels for Pulseaudio are `error`, `warn`, `notice`, `info` and `debug`. Supervisor exposes the Python Logger levels as follows:

```python
class LogLevel(str, Enum):
    """Logging level of system."""

    DEBUG = "debug"
    INFO = "info"
    WARNING = "warning"
    ERROR = "error"
    CRITICAL = "critical"
```

Afaik we can use bashism, how about:

```suggestion
declare log_level
declare -A log_levels
log_levels=( ["debug"]="debug" ["info"]="info" ["warning"]="warn" ["error"]="error" ["critical"]="error" )
log_level="${log_levels[$(bashio::supervisor.logging)]}"

exec pulseaudio --system --disallow-exit --exit-idle-time=-1 --disable-shm --log-level="${log_level}"
```

That said, it doesn't seem we have API access in the plugin, I get this:
```
[09:27:40] ERROR: Unknown HTTP error occured
```

For DNS we pass in a config json, should we do it that way here as well?

Thoughts @pvizeli ?

Reply:
Yeah, that is the issue. The plugins get setup before the API is up. Kinda chicken-egg trouble. A proper way to exchange data between plugin and supervisor would be great

Reply:
Ah shoot, didn't realize that. So we do need a config json then. So this requires two PRs, one to supervisor  so it writes out this config file and then one here to use it.